### PR TITLE
encd-6039 software cleanup

### PR DESCRIPTION
**Pull request by hitz**
Will need a big patch on demo for full testing.

### CODE REVIEW

File: src/encoded/schemas/changelogs/software.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
 ## Changelog for software.json
+### Schema version 11
```
Comment (bonitalam) [inline src/encoded/schemas/changelogs/software.md]: As the code is now, this doesn't require a whole new schema version and is just a minor change. If we are making the "software_type" and "used_by" properties required that would warrant a new schema version and corresponding upgrade.

File: src/encoded/schemas/software.json [hunk 2]
```diff
@@ -43,7 +43,14 @@
                     "quantification",
                     "transcript identification",
                     "transcriptome assembly",
-                    "variant annotation"
+                    "variant annotation",
```
Comment (bonitalam) [inline src/encoded/schemas/software.json]: Can we put the new enums in alphabetical order with the rest of the existing enums? Just makes it easier to navigate through the form.

File: src/encoded/schemas/changelogs/software.md [hunk 3]
```diff
@@ -1,4 +1,7 @@
...
+
+* Added several items to software_type enum; deprecated purpose field but did not remove.
```
Comment (bonitalam) [inline src/encoded/schemas/changelogs/software.md]: We should probably just list all the enums here for consistency.

**Review by bonitalam — CHANGES_REQUESTED** [review]
See comments. Also wondering if we are making any properties required are not?

File: src/encoded/schemas/changelogs/software.md (hunk 3 above)
Comment (hitz) [reply src/encoded/schemas/changelogs/software.md]: do you mean the CHANGES to the enums or just the additions.

Comment (bonitalam) [reply src/encoded/schemas/changelogs/software.md]: List just the new additions

File: src/encoded/schemas/software.json [hunk 4]
```diff
@@ -20,7 +20,7 @@
     ],
     "properties": {
         "schema_version": {
-            "default": "10"
+            "default": "11"
```
Comment (bonitalam) [inline src/encoded/schemas/software.json]: This should be 10.

**Review by bonitalam — CHANGES_REQUESTED** [review]

File: src/encoded/schemas/changelogs/software.md [hunk 5]
```diff
@@ -1,4 +1,7 @@
 ## Changelog for software.json
+### Minor changes since chema version 10
+
+* Added utility, proteomics, statistics, visualization, machine learning, integrated analysis, genetics to software_type.  Deprecated but did not remove purpose.
```
Comment (bonitalam) [inline src/encoded/schemas/changelogs/software.md]: Enums and properties should be italicized

**Review by bonitalam — CHANGES_REQUESTED** [review]

File: src/encoded/schemas/changelogs/software.md (hunk 5 above)
Comment (bonitalam) [reply src/encoded/schemas/changelogs/software.md]: Also should intregated analysis be integrative analysis? If so, need update here and in json file.

**Review by bonitalam — APPROVED** [review]

**Pull request merged**